client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -26,7 +33,21 @@ const useInteractionsTab = (props: useInteractionsTabInput) :  useInteractionsTa
     }
 
     const loadInteractions = () => {
-        // TODO: add loading from DB
+        axios.get(`/intersections/contactEvent/${epidemiologyNumber}`)
+            .then((result) => {
+                const fetchedContactEvents: InteractionEventDialogData[] = result.data.map((eventFromServer: any) => {
```

Comment:
Is it possible to create an `eventFromServer` interface which has a contacts field so you won't need to use map over the data?